ladybug_comfort/cli/utci.py

```diff
@@ -0,0 +1,364 @@
+# coding=utf-8
+"""Utility functions for calculating Universal Thermal Climate Index (UTCI).
```

Comment:
I feel like this kinda belongs in the `map` subpackage rather than the `cli` subpackage. Unless you disagree, I think we should put it there.

Reply:
I moved utci.py into the `map` subpackage.